about/pricing/comparison.md

```diff
@@ -0,0 +1,67 @@
+
+Our aim with 2i2c's service model is to strike a balance between **scalability** and **flexibility**, with the constraints that we operate **transparently** and **collaboratively** with our communities, by runing **open-source** and **community-driven** infrastructure.
+
+We curate and integrate this infrastructure, customize it for use-cases in research and education, and contribute back to the open source communities that underlie the tools we use.
+We use shared configuration and deployment infrastructure so that we can achieve economies of scale in serving many communities, but with an entirely open-source stack that can be customized for their unique needs.
+
+We do not know of any other organizations that offer managed cloud services in this way.
+However, below are three similar kinds of service and product models.
+They are roughly ordered from "most similar" to "least similar" to 2i2c.
+
+## Internal staffing
+
+The most common way for organizations to achieve similar services is to staff their own internal teams.
+2i2c encourages this, as it is aligned with our commitment to open source, vendor-agnostic tools, and the [Right to Replicate your infrastructure](https://2i2c.org/right-to-replicate).
+
+However, hiring and retaining modern cloud engineers is difficult and costly.
+If we assume that an engineer makes `$140,000` in salary, with `30%` benefits, that comes to an annual cost of `$182,000` a year, discounting any other personnel, hiring, and cloud costs.
+
+This under-estimates the true cost, as centralizing your organization’s cloud engineering on a single person creates risk associated with having a single point of failure.
+Their efficiency will depend heavily on their previous expertise, and they will likely not incorporate enhancement and security fixes as quickly as a distributed team of experts.
+
+If your organization has significant pre-existing expertise in open source, Jupyter, and cloud infrastructure, then it may be more cost effective for you to run your own JupyterHub services.
+If you need to build this expertise internally, it is likely much more cost-effective to partner with a non-profit such as 2i2c.
+
+## Consulting companies
+
+Many companies specialize in technical consulting that is flexible and tailored to an organization's needs.
+They can build bespoke infrastructure using an open source stack that is similar to the one that 2i2c offers.
+We also encourage this, as we believe in having a diverse ecosystem of vendors that can offer a similar vendor-neutral stack, as this may also encourage the [Right to Replicate your infrastructure](https://2i2c.org/right-to-replicate).
+Examples of companies that do software consultancy in this space are [Anaconda](https://www.anaconda.com/), [QuanSight](https://www.quansight.com/), and [QuantStack](https://quantstack.net/).
```

Comment:
Do you really want to be explicit about other companies' names? I understand that in an internal gdocs, but for a public website, I think it might be a little bit too much...

Reply:
Damian is wise here. Namechecking Anaconda but not mentioning Saturn Cloud or Coiled? I suggest that we omit explicit names of companies at this stage and develop a collective view on how 2i2c interacts with partner companies. We need to define criteria that provides guidance for which companies we can endorse, etc.

Reply:
I may be wrong on the "no names" advice in comment above. I see that more companies are listed below, including Google Colab. The transparency here is confident and consistent with 2i2c's open approach.

Reply:
I'll disagree with the comments--I think it's nice to frame 2i2c as part of an ecosystem of companies and consultants. These consulting companies are different from product companies like Coiled or Saturn.

Reply:
how about I add a note at the top of the page that notes we'll list similar services and companies, but in a way that is non-exhaustive and not an "endorsement"? I see 2i2c's role as a connector and an integrator in this space, so I think there is some value in providing visibility of other services so long as it doesn't seem like we are "picking favorites". I'll try adding this in the next commit and we can compare.